src/test/regress/sql/grant_on_schema_propagation.sql

```diff
@@ -0,0 +1,178 @@
+--
+-- GRANT_ON_SCHEMA_PROPAGATION
+--
+
+-- test grants are propagated when the schema is
+CREATE SCHEMA dist_schema;
+CREATE TABLE dist_schema.dist_table (id int);
+CREATE SCHEMA another_dist_schema;
+CREATE TABLE another_dist_schema.dist_table (id int);
+CREATE SCHEMA non_dist_schema;
+
+-- create roles on all nodes
+SELECT run_command_on_coordinator_and_workers('CREATE USER role_1');
```

Comment:
these users are kept on the cluster after this test. would it be possible to remove them so other tests could use the same names

Reply:
The roles are already dropped at the end of the test